native/packages/react-native-bpk-component-banner-alert/src/common-types.js

```diff
@@ -0,0 +1,87 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+
+import { type Node } from 'react';
+import PropTypes from 'prop-types';
+import { ViewPropTypes } from 'react-native';
+
+import {
+  dismissablePropType,
+  dismissableLabelPropType,
+  toggleExpandedButtonLabelPropType,
+} from './customPropTypes';
+
+export const ALERT_TYPES = {
+  success: 'success',
+  warn: 'warn',
+  error: 'error',
+  neutral: 'neutral',
+};
+
+export type Props = {
+  message: string,
+  type: $Keys<typeof ALERT_TYPES>,
+  animateOnEnter: boolean,
+  animateOnLeave: boolean,
+  children: Node,
+  dismissable: boolean,
+  dismissButtonLabel: ?string,
+  expanded: boolean,
+  onDismiss: ?(event: SyntheticEvent<>) => mixed,
+  onToggleExpanded: ?(event: SyntheticEvent<>) => mixed,
+  show: boolean,
+  toggleExpandedButtonLabel: ?string,
+  bannerStyle: ?any,
+};
+
+export const commonPropTypes = {
+  message: PropTypes.string.isRequired,
+  type: PropTypes.oneOf([
```

Comment:
This PropType does not match the Flow type. Flow specifies the keys of the `ALERT_TYPES` object whereas this uses the values. Now since they are the same this is not an immediate problem, but it's strange

Reply:
you could also use `Object.keys` like in the button component 
https://github.com/Skyscanner/backpack/blob/master/native/packages/react-native-bpk-component-button-link/src/common-types.js#L55

Reply:
Done!